### ISSUE TITLE
[IGNORE OLD] 26 params - corbensorenson + Codex (Track E self-contained)

### ISSUE THREAD

**Issue by corbensorenson** [anadim/AdderBoard#27]
### Author

corbensorenson + Codex

### Unique Parameter Count

26

### Accuracy

100.00% on 10,010 test cases (10 edge + 10,000 random, seed=2025)

### Method

Hand-coded weights

### Architecture

1L decoder, d=2, 1h, hd=2, tiny ReLU MLP, tied Q/K and tied V/O

### Key Tricks

- Factorized quadratic tied embeddings (`token_values + const0 + quad0`)
- One-head RoPE-19 routing for digit alignment
- Cross-tie attention `V/O` projection reused as MLP second projection
- Compressed MLP first projection via explicit scalar ties
- True parameter tying (`k_proj = q_proj`, `o_proj = v_proj`) with no value-dedup-only alias tricks
- No float buffer tables (no pair-sum/carry lookup table)

### Link to Code

https://github.com/corbensorenson/adderboard-submissions/blob/master/submissions/trackE.py

### Verification Output

```shell
$ python verify.py submissions/trackE.py --seed 2025
Model: Codex Track-E 26p TiedDense
Author: corbensorenson + Codex
Parameters (unique): 26
Architecture: 1-layer tied-dense decoder, d=2, 1h, hd=2, compressed MLP/embedding ties

Results: 10010/10010 correct (100.00%)
Status: QUALIFIED (threshold: 99%)
```

### Additional Notes

- Standard autoregressive decode loop in `add()` (append argmax next token each step).
- `forward()` is tensor-in/logits-out only; no digit loops / explicit carry variable / string logic.
- No direct `a+b` shortcut in submission code.
- Local conservative count check:
  - params (alias tying): 26
  - float buffers: 0
  - effective (params + float buffers): 26

(Prepared with Codex.)
